markdown/boomi-v2-getting-started/boomi-v2-getting-started.md

```diff
@@ -161,54 +158,37 @@ Select your newly created environment and search for your atom to attach it
 Positive
 : Note: a blue icon next to your newly created Atom means that it has an Online Status
 
```

Comment:
I can only comment on lines next to changes. This is for line 67. 
I believe an 's' is needed at the end of 'manages'. 

> Boomi builds, deploys and manages

Reply:
aah good catch, i'll update 👍